src/main/resources/swagger/agora.yaml

```diff
@@ -148,6 +149,38 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+    delete:
+      summary: Redact a single method.
+      description: |
+        Redacts a method and all of its associated configurations
+      parameters:
+        - name: namespace
+          in: path
+          description: Namespace of method.
+          required: true
+          type: string
+        - name: name
+          in: path
+          description: Name of method.
+          required: true
+          type: string
+        - name: snapshotId
+          in: path
+          description: Snapshot ID of method.
+          required: true
+          type: number
+      tags:
+        - Methods
+      responses:
+        200:
+          description: |
+            An integer showing how many methods were deleted.
+          schema:
+            type: number
+        default:
```

Comment:
Throughout, we have responses listed as "default", where we should probably be specifying their type (500, 400, etc).

Also, there are some cases where we have 400 responses, but #/definitions/Error only has 500. We should probably create some more error response definitions. Might as well do this now, while we're editing the file?